garden_ai/app/model.py

```diff
@@ -0,0 +1,58 @@
+from pathlib import Path
+from typing import Optional, List
+
+from garden_ai.client import GardenClient
+
+import typer
+import rich
+
+model_app = typer.Typer(name="model", no_args_is_help=True)
+
+
+@model_app.callback()
+def pipeline():
+    """
+    sub-commands for managing machine learning models
+    """
+    pass
+
+
+@model_app.command(no_args_is_help=True)
+def register(
+    name: str = typer.Argument(
+        None,
```

Comment:
heads up - typer reads `None` as "this is optional" and `...` as "required" when it sees either as a default https://typer.tiangolo.com/tutorial/arguments/optional/

Reply:
Ah, thanks! Definitely want it to be required